core/templates/pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -129,6 +136,20 @@ export class ContributionOpportunitiesBackendApiService {
         params, resolve, reject);
     });
   }
+
+  async fetchFeaturedTranslationLanguages(): Promise<Object> {
+    const response = await this.http
```

Comment:
@bansalnitish, do we follow async/await pattern for the services?

Reply:
Note - @sagangwee suggested that we start using async/await over promise chaining above

Reply:
No, for the services I migrated or reviewed I didn't see this. What are your thoughts @srijanreddy98 ?

Reply:
@DubeySandeep, @bansalnitish, and @sagangwee   We haven't used async-await in any of the services migrated yet. We don't have that sort of a pattern in services.
I think it would be better if we didn't introduce such a change now, as many of the services have been migrated and we would want to keep a uniform coding style.
That said, I am not against the idea of using async-await. In fact, I would prefer to have them over `promise.then()` in the directives/ components when we start migrating them. If it is used, then it should be done properly. The first thing to keep in mind is that async functions can only return promises. Secondly, there should be a catch for each await we use. Something like:
```
aysnc f() {
const result = await someAsyncTask().catch(err => {
  ...
});
```
or one could simply use try...catch.